tool/src/test/java/org/wildfly/security/tool/MaskCommandTest.java

```diff
@@ -63,9 +64,9 @@ public void testMissingSaltAndIteration() {
         String retVal = executeCommandAndCheckStatusAndGetOutput(args);
         String[] retValLines = retVal.split(LINE_SEPARATOR);
 
-        assertTrue("Message about invalid salt parameter must be present", retValLines[0].contains("Invalid \"salt\" parameter. Generated value"));
-        assertTrue("Message about invalid iteration parameter must be present", ("Invalid \"iteration\" parameter. Default value \"" + defaultIteration + "\" will be used.").equals(retValLines[1]));
-        assertTrue("Message about invalid salt parameter must be present", retValLines[2].contains("MASK-"));
+        assertEquals("Message about invalid salt parameter must be present","Invalid \"salt\" parameter. Generated value", retValLines[0]);
```

Comment:
i think this test will still fail because `retValLines[0]` does not equal `"Invalid \"salt\" parameter. Generated value"`, but rather it just contains it as a substring